pkg/operator/config.go

```diff
@@ -47,11 +48,12 @@ type Config struct {
 }
 
 type ReloaderConfig struct {
-	CPURequest    string
-	CPULimit      string
-	MemoryRequest string
-	MemoryLimit   string
-	Image         string
+	CPURequest      string
+	CPULimit        string
+	MemoryRequest   string
+	MemoryLimit     string
+	Image           string
+	ImagePullPolicy v1.PullPolicy
```

Comment:
```suggestion
```

Reply:
Is this intentional to not be able to set imagePullPolicy for config-reloader?

Reply:
Actually it is being set but that function.

Reply:
Ah I see the misunderstanding now. See my comment above.